src/less/success-message/base/success-message.less

```diff
@@ -0,0 +1,67 @@
+.success-message__button,
+.success-message__fake-button {
+    .btn-base-no-font-size();
+
+    background-color: @success-message-button-color;
+    border-color: @success-message-button-border-color;
+    font-size: @font-size-16;
+    margin-top: 28px;
+    min-height: 48px;
+    padding: 15px 16px 13px;
+
+    &:hover {
+        background-color: @success-message-button-color-hover;
+        border-color: @success-message-button-color-hover;
+        color: @success-message-button-border-color;
+        text-decoration: none;
+    }
+}
+
+.success-message--full {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    position: relative;
+
+    .success-message__button,
+    .success-message__fake-button {
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        width: 100%;
+    }
+}
+
+.success-message {
+    background-color: @success-message-background-color;
+    box-sizing: border-box;
+    padding: 0 9%;
+    text-align: center;
+    width: 100%;
+}
+
+.success-message__status {
+    height: 35px;
+    margin: 0 auto;
+    width: 35px;
+
+    span {
```

Comment:
Sorry one more nit!
Why are we having a span as an empty tag. Could we add the icon on the `success-message__status` class itself? 
If we want to keep styles for inline icons, maybe we could add another modifier class to show the icon, like `success-message__icon` for background icon. 
I just really don't like the empty `span` inside the `div`
Thoughts?
@ianmcburnie what do you think? 
Otherwise looks good to me

Reply:
I agree, could we just have a single span or div for the icon?

Reply:
Yeah, I didn't like this either. I was trying to follow the lead of notice which has an empty `h2` containing an empty `span`.  But I guess the difference is that has an aria-label.

So @agliga @ianmcburnie  are you OK with:
```
<div class="success-message__status success-message__status--inline"></div>
```

Reply:
But this becomes the most common (background icon) use case.

Reply:
Example:
Background icon would be:
```
<div class="success-message__status success-message__status--icon"></div>
```

Foreground icon would be:
```
<div class="success-message__status">
    <svg aria-hidden="true" focusable="false" height="35" width="35">
        <use xlink:href="#icon-confirmation-filled"></use>
    </svg>
</div>
```

Reply:
I like it. I think it would be good to change the other ones too eventually and get rid of the nested span in notice (but maybe with the next major version)

Reply:
Or I go back to just putting it as the background of the title itself, and we skip the foreground icon, since it was accessible and they shouldn't have any need to change the icon color. 😜

Reply:
Can it just be `<span class="success-message__status"></span>` or `<svg class="success-message__status"></span>`, without any modifier? That's usually how we do it elsewhere.

I think I need to revisit the Page Notice markup in skin v10, it's perhaps just a little too odd having the image as the h2. I could go into the reason for it, but it's far too long winded and I think it's not a factor anymore...

Reply:
I ended up removing the class from the SVG completely.  Take a look now.  Its almost what you said, but I'm using a div since it needs to be a block of some sort to get height and width.  Let me know if there is a benefit to using a `span` and I'll make that change.